src/persist-client/src/cfg.rs

```diff
@@ -416,6 +419,27 @@ pub const TXN_USE_CRITICAL_SINCE: Config<bool> = Config::new(
     "Use the critical since (instead of the overall since) when initializing a subscribe.",
 );
 
+/// Migrate the txns code to use the critical since when opening a new read handle.
+pub const CATALOG_USE_CRITICAL_SINCE: Config<bool> = Config::new(
+    "persist_catalog_use_critical_since",
+    false,
+    "Use the critical since (instead of the overall since) for the Persist-backed catalog.",
+);
+
+/// Migrate the txns code to use the critical since when opening a new read handle.
+pub const SOURCE_USE_CRITICAL_SINCE: Config<bool> = Config::new(
+    "persist_source_use_critical_since",
+    true,
```

Comment:
Is this default intentionally true? I'm guessing yes given that it's omitted from `__init__.py` but I don't see where we've explained why this one is different.

Reply:
IIRC I thought this one felt more like the `txn` case than the `catalog` case - well-understood enough to feel comfortable defaulting to true. But I feel extremely weakly about it and am happy to do whatever you prefer! (And I should have called this out either way - thanks for mentioning it.)

Reply:
Your call! I just wanted ☝️ this justification written down somewhere.

Reply:
Aight - defaulted to false, and I'll plan on rolling it out before the other flags as a "practice run" I guess!